fx-src/index.js

```diff
@@ -18,11 +18,9 @@
 import KintoClientBase from "../src/base";
 import * as errors from "../src/errors";
 
-const Cu = Components.utils;
-
-Cu.import("resource://gre/modules/Timer.jsm");
+ChromeUtils.import("resource://gre/modules/Timer.jsm");
```

Comment:
I didn't know :) 

https://bugzilla.mozilla.org/show_bug.cgi?id=1425611

Reply:
Yeah, it's a bit of a surprise, but an announcement was posted to dev-platform/firefox-dev.